canvas_account_admin_tools/views.py

```diff
@@ -76,10 +76,13 @@ def dashboard_account(request):
         custom_canvas_account_id
     )
 
+    # todo: confirm what conditions need to be met for course_info to be available
+    course_info = True
     manage_account = [
         conclude_courses,
         lti_tools_usage,
-        courses_in_this_account
+        courses_in_this_account,
+        course_info
     ]
 
     return render(request, 'canvas_account_admin_tools/dashboard_account.html', {
```

Comment:
`has_manage_account` should probably just be `any(manage_account)`...no need to pass in all the tool objects.